src/components/ReleaseList/ReleaseList.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {Header, Loader, Comment, Icon, Image, Table} from 'semantic-ui-react';
+import Markdown from 'react-markdown';
+import {longDate} from '../../common/dateUtils';
+
+const ReleaseList = ({loading, releases}) => {
+  const coordUrl = process.env.REACT_APP_COORD_UI + 'releases/';
+
+  if (loading) {
+    return <Loader>Loading...</Loader>;
+  }
+
+  if (!loading && releases.length === 0) {
+    return (
+      <Header disabled textAlign="center" as="h4">
+        No releases data available for current study.
+      </Header>
+    );
+  }
+
+  return (
+    <Comment.Group size="large">
+      {releases.map(r => (
+        <Comment key={r.node.id}>
+          <Comment.Avatar as={Icon} name="tag" />
+          <Comment.Content>
+            <Comment.Author
```

Comment:
Can we make this appear as a blue link with an external icon next to it to stay consistent with our external url formatting?